configuration/rules-blockly-timers-and-delays.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: documentation
+title: Rules Blockly - Timers & Delays
+---
+
+# Timers & Delays
+[return to Blockly Reference]({{base}}/configuration/rules-blockly.html)
+
+Timers and Delays are a little more complex but add important functionality to rules. Whilst the "Wait-For"-block is straightforward, the timer blocks should be well understood before being used - they may behave differently than expected. This chapter explains what these blocks do, sometimes displaying generated code to explain what is happening behind the scenes.
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+{: #blockly-timers-and-delays-overview}
+
+## Overview of the Timers and Delays category
+
+> ![timers-and-delays](images/blockly/blockly-timers-and-delays-1.png) ![timers-and-delays.png](images/blockly/blockly-timers-and-delays-2.png)
+
+##Timer Naming
+
+Timers are created and referred to by name, enabling manipulation within a rule.
+
+> **Important**: a named timer is *only* available within the same rule. The same timer *cannot* be accessed via a different rule. A different rule with a timer of the same name results into two separate timers.
+
+### Wait for
+
+![waitfor](images/blockly/blockly-waitfor.png)
+
+*Function:* Suspends execution of the rule for a given period of time
+
+- unit of time (ms) is milliseconds (1000 ms = 1 second)
+
+*Example*
+
+The following simple example uses a loop to implement a blinking light with a 1 second delay, looping three times:
+
+![waifor-example](images/blockly/blockly-waitfor-example.png)
+
+### After *period of time* Do With Timer
+
+![blockly-afterperiodtimer](images/blockly/blockly-afterperiod-timer.png)
+
+*Function:* Schedule a task to be performed once after a specific amount of time has elapsed.
+
+- A number and unit (seconds, minutes up to months) can be chosen after which the commands within the block will be executed
+- Provide the name of the timer, allowing further access to that timer within the same rule. The name also ensures the timer is unique within the rule.
```

Comment:
Please change this to one sentence per line.
The website build will take care of a proper presentation of paragraphs.